source/dub/commandline.d

```diff
@@ -562,8 +562,8 @@ struct CommonOptions {
 	bool help, annotate, bare;
 	string[] registry_urls;
 	string root_path;
-	enum colors { automatic, on, off } // Style violation in support of invalid option error formatting.
-	colors colors_mode = colors.automatic;
+	enum Color { automatic, on, off } // Style violation in support of invalid option error formatting.
```

Comment:
By changing the case from `color` to `Color` it is no longer a style violation ;-)

The reason I chose to violate the style and use lower case is so that if a user selects an unsupported value (like `--color=blue`) a corresponding error message is produced to the effect of "`blue` is not a valid value for `color`". With your change the error will become "`blue` is not a valid value for `Color`", which is not consistent with the option name.

Reply:
Perhaps the comment should read `// Lower case "color" in support of invalid option error formatting.` instead...

Reply:
Adjusted via https://github.com/dlang/dub/compare/a9a740c0d0985ed86e77dae7c06175d576927f78..53b01ca3d6e10a742c5c7790f648474120dccb82.